<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandleData</class>
 <widget class="QMainWindow" name="HandleData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="PushButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReturnButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>撤销修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>添加记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>删除选中行</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="LineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>01月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11月</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12月</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FindButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>查找数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
